engine/gamesys/src/gamesys/scripts/script_label.cpp

```diff
@@ -264,7 +264,7 @@ static const char* LABEL_EXT = "labelc";
  *
  * @name label.set_text
  * @param url [type:string|hash|url] the label that should have a constant set
- * @param text [type:string] the text
+ * @param text [type:string|number] the text
```

Comment:
I think it's perhaps more correct to say anything that supports "tostring()".
Perhaps `object` is the way to go here?

Reply:
> I think it's perhaps more correct to say anything that supports "tostring()".
> Perhaps `object` is the way to go here?

I don't think we are documenting it like this anywhere else?

Reply:
bad argument #2 to 'set_text' (string expected, got boolean)
bad argument #2 to 'set_text' (string expected, got table)

number and string worked